widget/embedded/src/utils/wallets.ts

```diff
@@ -568,3 +484,40 @@ export function getAddress({
       connectedWallet.chain === chain
   )?.address;
 }
+
+export function makeTokensBalance(connectedWallets: ConnectedWallet[]) {
```

Comment:
I think you didn't consider that a user could have different balance for a single token on different wallets. We should consider that.

Reply:
Oh it seems you did it as we only show the max balance for user. 
But if we want to show balance of each token per each wallet in our dapp through our dapp, it it's using this store, we should change it. If not, it's okay.